examples/claude-try-it/install.ps1

```
@@ -0,0 +1,798 @@
+        }
+    }
+}
+
+# --- Find a free port -----------------------------------------------------
+
+function Find-FreePort {
+    foreach ($port in 5432, 5433, 5434, 5435, 5436) {
+        $inUse = $null
+        try {
+            $inUse = Get-NetTCPConnection -LocalPort $port -ErrorAction SilentlyContinue
+        } catch {}
```

PSScriptAnalyzer (1.24.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)